### ISSUE TITLE
Add selectable updates

### ISSUE THREAD

**Issue by Tecnico1931** [patchoo/patchoo#14]
Sample:

https://github.com/mm2270/CasperSuiteScripts/blob/master/selectable_SoftwareUpdate.sh

**Comment by charliwest** [comment]
I think looking at how homebysix's auto update magic installs updates in the back ground as long as certain apps aren't running and merging that into patchoo would be a great partial implementation of this feature

**Comment by loceee** [comment]
I agree, but without extra pkg metadata to specify the process that blocks and installation it can't really work. Defining this as an extra parameter for each pkg cache policy "might" be a solution. But I have bigger plans to solve all this with patchoo2.

I am also waiting to see what jamf announce tomorrow before I go too much further.
